app/src/main/java/com/puzzlebench/clean_marvel_kotlin/domain/model/Character.kt

```diff
@@ -1,7 +1,39 @@
 package com.puzzlebench.clean_marvel_kotlin.domain.model
 
+import android.os.Parcel
+import android.os.Parcelable
+
+
 class Character(
+        val id: Int,
         val name: String,
         val description: String,
         val thumbnail: Thumbnail
-)
+) : Parcelable {
+    constructor(parcel: Parcel) : this(
+            parcel.readInt(),
+            parcel.readString(),
+            parcel.readString(),
+            TODO("thumbnail"))
+
+    override fun writeToParcel(parcel: Parcel, flags: Int) {
+        parcel.writeInt(id)
+        parcel.writeString(name)
+        parcel.writeString(description)
+    }
+
+    override fun describeContents(): Int {
+        return 0
```

Comment:
Pls not use hardcodes. put into a componion obj as a private const

Reply:
if you use a const, give it a representative name